### PR TITLE
Add missing methods and iDEAL issuers

### PR DESCRIPTION
**Pull request by Smitsel**
Added some missing methods:
- ING Home'pay (https://www.mollie.com/en/payments/ing-homepay)
- Paypal (https://www.mollie.com/en/payments/paypal)
- Paysafecard (https://www.mollie.com/en/payments/paysafecard)
- Bitcoin (https://www.mollie.com/en/payments/bitcoin)

Also corrected ``Bunq`` to ``bunq``.
Added ``Moneyou`` iDEAL issuer.
And squashes the test and live issuer lists together, since they are the same.

### CODE REVIEW

File: src/Resources/config/services.xml [hunk 1]
```diff
@@ -14,27 +14,20 @@
...
-            <parameter key="ideal_SNSBNL2A">SNS Bank (test)</parameter>
-            <parameter key="ideal_TRIONL2U">Triodos Bank (test)</parameter>
-            <parameter key="ideal_FVLBNL22">Van Lanschot (test)</parameter>
-        </parameter>
-        <parameter type="collection" key="ruudk_payment_mollie.ideal.issuers.live">
+        <parameter type="collection" key="ruudk_payment_mollie.ideal.issuers">
             <parameter key="ideal_ABNANL2A">ABN AMRO</parameter>
             <parameter key="ideal_ASNBNL21">ASN Bank</parameter>
-            <parameter key="ideal_BUNQNL2A">Bunq</parameter>
+            <parameter key="ideal_BUNQNL2A">bunq</parameter>
```
Comment (ruudk) [inline src/Resources/config/services.xml]: Can you please revert this line? I really don't like this. Every brandname should start with a capital.

Comment (ruudk) [reply src/Resources/config/services.xml]: People that want to have it starting with a lowercase b are free to change that / overwrite it.

**Comment by Smitsel** [comment]
Allright! Changed the label back to ``Bunq``

**Comment by ruudk** [comment]
Tagged https://github.com/ruudk/PaymentMollieBundle/releases/tag/6.2.0

**Pull request merged**